src/fairseq2/recipes/lm/instruction_finetune.py

```diff
@@ -113,7 +113,7 @@ class InstructionFinetuneConfig:
     num_lr_warmup_steps: int = 1  # TODO: 2000
     """The number of learning rate warm-up steps."""
 
-    gradient_accumulation: int = 1
+    gradient_accumulation: int = 2
```

Comment:
Is this intentional?

Reply:
yeah I figured that its always good to have that batch order randomness in by default when we have fixed content pre-bucketed batches
no strong opinion though